content/docs/faq-structure.md

```diff
@@ -59,16 +59,16 @@ components/
   ProfileHeader.css
 ```
 
-Some people also prefer to go further, and separate components into different folders depending on their role in the application. For example, [Atomic Design](http://bradfrost.com/blog/post/atomic-web-design/) is a design methodology built on this principle. Remember that it's often more productive to treat such methodologies as helpful examples rather than strict rules to follow.
+ישנם אנשים שמעדיפים ללכת רחוק יותר, ולהפריד קומפוננטות לתוך תיקיות על פי התפקיד שלהן ביישום. לדוגמה, [Atomic Design](http://bradfrost.com/blog/post/atomic-web-design/) היא מתודולוגיית עיצוב שנבנתה על עיקרון זה. זכור שזה לעיתים יותר פרודוקטיבי לנהוג במתודולוגיות כאלה כדוגמות שיכולות לעזור מאשר כחוקים שצריך לציית להם.
 
-#### Avoid too much nesting {#avoid-too-much-nesting}
+#### הימנע מיותר מדי nesting {#avoid-too-much-nesting}
 
-There are many pain points associated with deep directory nesting in JavaScript projects. It becomes harder to write relative imports between them, or to update those imports when the files are moved. Unless you have a very compelling reason to use a deep folder structure, consider limiting yourself to a maximum of three or four nested folders within a single project. Of course, this is only a recommendation, and it may not be relevant to your project.
```

Comment:
What do you think of ייבוא יחסי as a translated word for relative imports?

Reply:
Not sure about that, I think it might create more confusion